backend/route/views.py

```diff
@@ -1,3 +1,269 @@
-#from django.shortcuts import render
+from django.http import HttpResponse, HttpResponseNotAllowed, JsonResponse,HttpResponseBadRequest
+from django.contrib.auth.models import User
+from django.views.decorators.csrf import ensure_csrf_cookie
+import json
+from .models import *
+from json.decoder import JSONDecodeError
 
-# Create your views here.
+def posts(request):
+    if request.method!='GET' and request.method!='POST':
```

Comment:
How about using `@require_http_methods` decorator?
It would be more clear what methods are used for each view function.

Here is the reference about this decorator.
https://docs.djangoproject.com/en/3.2/topics/http/decorators/